packages/arb-avm-cpp/cmachine/storage.go

```diff
@@ -45,16 +45,23 @@ func boolToCInt(b bool) C.int {
 	return C.int(x)
 }
 
+func stringToPruningMode(mode string) C.CPruningMode {
```

Comment:
Would be good to have an error check here if mode is an invalid value so that we catch it if the config check deviates from this

Reply:
:+1: